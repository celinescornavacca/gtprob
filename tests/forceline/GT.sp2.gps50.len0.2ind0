(((((((A12,((A42,A43),A13)),(A2,((A10,((A23,((A48,A49),(A40,A41))),A11)),A6))),(((((A21,A22),((A15,A16),A8)),(A24,(A39,(A46,A47)))),((A30,A31),A3)),A0)),((A32,A33),(A4,A5))),((A1,(A7,((A17,A18),((((A35,A36),A34),(A25,A26)),A9)))),((A19,A20),(((A29,(A37,A38)),A14),(A44,A45))))),(A27,A28)),((((B3,(B28,B29)),(((B18,B19),((B46,B47),(B42,B43))),(((B13,(B44,B45)),(B40,B41)),B6))),(((B7,B8),(B21,B22)),(((B25,B26),B14),((B20,((B32,B33),B31)),B17)))),(((B0,(B1,((B48,B49),B15))),((((((B23,B24),(B27,(B38,B39))),(B36,B37)),B5),(B9,B10)),B2)),((B11,B12),((((B34,B35),B30),B16),B4)))));
