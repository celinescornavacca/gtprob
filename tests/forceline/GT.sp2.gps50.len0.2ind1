(((((A6,((((A44,A45),A41),A22),A9)),((A5,((A16,A17),A10)),(((A2,(A39,A40)),(A7,(A20,A21))),(A3,(A25,A26))))),((((A31,A32),(A33,A34)),(A19,(A42,A43))),(A29,A30))),((((A23,A24),(A46,A47)),(((A12,A13),((A27,A28),A18)),A8)),((A1,(A4,((A14,A15),A11))),(((A48,A49),(A35,A36)),((A37,A38),A0))))),(((B1,(((B12,(B30,(B48,B49))),(B18,B19)),(((B24,B25),((B14,B15),B13)),((B40,B41),(B7,B8))))),(((B6,(B20,(B34,B35))),((((((B21,B22),B5),((B4,((B26,B27),B23)),(B3,(B42,B43)))),(((B44,B45),(B32,B33)),(B9,(B38,B39)))),(B36,B37)),(((B16,B17),B11),B2))),((B31,(B46,B47)),((B28,B29),B10)))),B0));
