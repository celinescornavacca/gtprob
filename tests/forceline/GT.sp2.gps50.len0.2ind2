((((A12,(A20,A21)),(A38,A39)),((((A9,A10),((((A34,A35),A25),(A42,A43)),(A44,A45))),(A0,(A8,(((A30,A31),A24),A11)))),((((A2,(A13,A14)),(A15,(A16,A17))),(((A28,A29),(A6,A7)),((A4,(A5,(A36,A37))),((A22,A23),(A46,A47))))),(A1,(((A18,((A40,A41),(A32,A33))),((A26,A27),(A19,(A48,A49)))),A3))))),((((B0,(((B13,((B46,B47),B35)),(B7,(B8,B9))),((B38,B39),B29))),((B5,(B11,B12)),((B48,B49),B16))),((((B23,(B27,B28)),B17),B3),(B1,((B40,B41),(B36,B37))))),(((B6,(B22,(B32,B33))),((B19,(((B42,B43),B34),B31)),(B18,(B20,B21)))),(((B24,B25),B4),(((B10,(((B44,B45),B30),B26)),(B14,B15)),B2)))));
