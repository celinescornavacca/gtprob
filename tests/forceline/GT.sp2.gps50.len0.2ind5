((((A37,A38),(((A8,A9),A2),((((A28,A29),A23),(A21,A22)),(((A14,A15),(A5,(A13,(A20,(A26,A27))))),(A19,(A24,(A48,A49))))))),((((A45,(A46,A47)),(A35,A36)),(((A41,A42),(((A6,A7),((A12,(A32,A33)),A4)),A0)),(A16,((A34,(A43,A44)),A25)))),((A10,A11),(A1,(((A18,(A30,A31)),A3),(A17,(A39,A40))))))),((((((((B15,B16),B7),(B21,B22)),(B44,B45)),(((B35,B36),B3),((B12,B13),((B14,(((B40,B41),B39),B23)),((B8,(B9,(B17,(B27,B28)))),(B19,B20)))))),(((B24,(B48,B49)),((B25,B26),((B37,B38),B29))),(((B6,((B33,B34),B32)),B5),(B10,B11)))),(((B30,B31),B2),B1)),(((((B46,B47),(B42,B43)),B18),B4),B0)));
