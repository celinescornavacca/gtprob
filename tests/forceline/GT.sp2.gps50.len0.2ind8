(((A0,(A1,((A12,(A41,A42)),((A46,A47),A43)))),((((((A34,(A37,A38)),A26),((A9,A10),A5)),(((A44,A45),((A39,A40),(A27,A28))),(A35,A36))),(A11,(((A32,A33),(A48,A49)),A15))),((A20,A21),((((A7,A8),(A13,A14)),(A3,A4)),((A2,(A6,(A22,A23))),((((A16,A17),(A29,(A30,A31))),(A24,A25)),(A18,A19))))))),((B0,(((B8,(((B30,B31),B13),(B32,B33))),((B19,(B23,B24)),B15)),(B2,(((B3,((B22,(B42,B43)),B5)),(((B25,((B44,B45),(B40,B41))),(B9,B10)),((B28,(B38,B39)),(B26,B27)))),(B6,B7))))),(((B4,((B48,B49),(B34,B35))),((((B16,B17),(((B46,B47),B29),B14)),B1),((B20,B21),(B11,B12)))),(B18,(B36,B37)))));
