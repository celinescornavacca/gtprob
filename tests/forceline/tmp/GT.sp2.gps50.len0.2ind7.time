0.47user 0.10system 0:00.78elapsed 73%CPU (0avgtext+0avgdata 131264maxresident)k
16inputs+8outputs (0major+14180minor)pagefaults 0swaps
