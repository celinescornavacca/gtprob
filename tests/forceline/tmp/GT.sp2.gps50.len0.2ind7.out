-343.261073961
